src/app/onion/dashboard/dashboard.component.html

```diff
@@ -1,3 +1,40 @@
-<p>
-  dashboard works!
-</p>
+<div class="dashboard">
+  <clark-dashboard-splash
+    (navigateBack)="navigateBack()"
+  ></clark-dashboard-splash>
+  <div class="dashboard__lists">
+    <div class="dashboard__tabs">
+      <button
```

Comment:
Aria-label please. This should also receive the corresponding aria-roles because they are tabs. Here is a link to a better explanation https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role